src/app/screens/Onboard/ChooseConnector/index.tsx

```diff
@@ -63,7 +70,7 @@ export default function ChooseConnector({ title, description }: Props) {
             </p>
           )}
         </div>
-        <div className="grid grid-cols-5 gap-5">
+        <div className="grid grid-cols-4 gap-5">
```

Comment:
why did you change it to be 4 columns? Alby should be in the first line - it is also the easiest to get started.

Reply:
We used to have it at 4 columns, I thought we might want to change it back? If not I can remove this.